tests/test_types.py

```diff
@@ -85,13 +85,14 @@ def combine_typed_rapameters(spec_seq: Iterable[Tuple[SaType, Iterable]]):
     (t.Float32, [0., -0., 1e-37, -1e-37, 1e38, -1e38]),
     (t.Float64, [0., -0., 1e-307, -1e-307, 1e308, -1e308]),
 
-    (t.Decimal(9, 4), map(Decimal, [0, '1.2345', '-1.2345'])),
```

Comment:
`Argument 1 to "map" has incompatible type "Type[Decimal]"; expected "Callable[[object], Any]"`

Looks like a bug in mypy